cpp/include/Ice/Exception.h

```diff
@@ -15,9 +15,16 @@ namespace Ice
     class ICE_API Exception : public std::exception
     {
     public:
+        /// Default constructor.
         Exception() noexcept = default;
-        Exception(const Exception&) noexcept = default;
-        Exception& operator=(const Exception&) noexcept = default;
+
+        /// Copy constructor. Initializes this exception with a copy of other.
```

Comment:
Is Doxygen smart enough to see that `other` has an `@param` tag underneath, or should we have:
```suggestion
        /// Copy constructor. Initializes this exception with a copy of @p other.
```